### DISCUSSION TITLE
Is there a working example of using Facade API?

### DISCUSSION THREAD

**Discussion by salmoro** [dream-num/univer#1148] [Q&A | 常见问题]
When I try using as mentioned in the docs I get a redi error:
```
Uncaught (in promise) Error: [redi]: Cannot find "formula-ui.register-function-service" registered by any injector.
```
In the source code it seems like this dependency is added to the injector so maybe I'm doing something wrong in the setup.

I'm on the latest version of `0.1.0-beta.2`

**Comment by jikkai** [comment]
Here is the documentation about facade: https://univer.ai/en-us/guides/facade/
If you want to get more help, please provide a minimal reproduction.
